### PR TITLE
created workflows, additional file mods

### PR DESCRIPTION
**Pull request by dalerichardson**

### CODE REVIEW

**Review by lorengordon — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    paths:
+      - .bumpversion.cfg
+
+jobs:
+  release:
+    uses: plus3it/actions-workflows/.github/workflows/release.yml@41056300ba78d8be98b45f057aea246f8c4f30cf
+    with:
+      mockstacktest-enable: false
+    secrets:
+      release-token: ${{ secrets.GH_RELEASES_TOKEN }}
```
[the remark this answers is not in the document]
Comment (dalerichardson) [reply .github/workflows/release.yml:20]: When I pushed it to my fork, the mockstacktest failed. See - https://github.com/plus3it/terraform-aws-tardigrade-config-rules/actions/runs/4115735014/jobs/7104850693

[the remark this answers is not in the document]
Comment (dalerichardson) [reply .github/workflows/release.yml:20]: Then I added the disable mockstack test logic

[the remark this answers is not in the document]
Comment (lorengordon) [reply .github/workflows/release.yml:20]: Yeah, that's a weird error though, just saying it cannot connect. Might have been a race condition. It is working now, so we can remove it and make that a required workflow.

[the remark this answers is not in the document]
Comment (dalerichardson) [reply .github/workflows/release.yml:20]: Sounds good. Will remove disable logic.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+on:
+  pull_request:
+
+concurrency:
+  group: lint-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    uses: plus3it/actions-workflows/.github/workflows/lint.yml@41056300ba78d8be98b45f057aea246f8c4f30cf
```
[the remark this answers is not in the document]
Comment (dalerichardson) [reply .github/workflows/lint.yml:11]: Can you clarify? You are referring to the @sha value we set here?

[the remark this answers is not in the document]
Comment (lorengordon) [reply .github/workflows/lint.yml:11]: Yes, the sha value. The current value is for v1.0.0 of the actions-workflows project. We released v1.0.1 on Monday. So these values need to be updated to the sha for v1.0.1.

[the remark this answers is not in the document]
Comment (dalerichardson) [reply .github/workflows/lint.yml]: Got it, thanks

**Review by lorengordon — APPROVED** [review]
Looks good!

**Pull request merged**